tools/util/marshaller.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+// this file directly copy/pasted from KubeVirt project here:
+// https://github.com/kubevirt/kubevirt/blob/master/tools/util/marshaller.go
+
+package util
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// MarshallObject marshalls an object to yaml appropriate for kubectl
+func MarshallObject(obj interface{}, writer io.Writer) error {
```

Comment:
where will this be used?

Reply:
was required for installStrategy. Now removed, since this approach was not taken